addon/popup.js

```diff
@@ -1195,7 +1195,7 @@ function getRecordId(href) {
   }
 
   // Lightning Experience and Salesforce1
-  if (url.hostname.endsWith(".lightning.force.com")) {
+  if (url.hostname.endsWith(".lightning.force.com") || url.hostname.endsWith(".lightning.force.com.us.cas.ms")) {

```

Comment:
This seems pretty custom. I would like to suggest that the extension has some configuration that the user could add this domain extension to.  I am new to chrome extensions but I am pretty sure that can be done.